docs/README.md

```diff
@@ -6,18 +6,21 @@ If you have Tinkerbell running in your environment, you can use it for CAPT.
 
 Here is the list of required components to use CAPT:
 
-- Existing Tinkerbell installation running at least versions mentioned in [sandbox](https://github.com/tinkerbell/sandbox/commit/be40a7b371fdab947a413e190cfe99b692651d8e).
+- Existing Tinkerbell installation running at least versions mentioned in [sandbox](https://github.com/tinkerbell/sandbox/tree/v0.5.0), this guide assumes deployment using the sandbox with an IP address of 192.168.1.1, so modifications will be needed if Tinkerbell was deployed with a different method or if the IP address is different.
 - A Kubernetes cluster which pods has access to your Tinkerbell instance.
-- A Container registry reachable by both your Kubernetes cluster and your workstation with
-  write access.
-- At least one Hardware available with DHCP IP address configured on first interface and with
-  /dev/vda disk.
+- At least one Hardware available with DHCP IP address configured on first interface and with proper metadata configured
 - `git` binary
 - `tilt` binary
 - `kubectl` binary
 - `clusterctl` binary
 - `go` binary
 
+Required metadata for hardware:
+- metadata.facility.facility_code is set (default is "onprem")
+- metadata.instance.id is set (should match the hardware's id)
+- metadata.instance.hostname is set
+- metadata.instance.storage.disks is set and contains at least one device matching an available disk on the system
+
 ### Workaround for 8000 bytes template limit
```

Comment:
This can be removed since we removed events from tink-server https://github.com/tinkerbell/tink/pull/468

Reply:
Yeah, was planning on updating the docs (and implementation) to reflect that once there is a sandbox release that includes that PR